pennylane/devices/default_clifford.py

```diff
@@ -0,0 +1,712 @@
+            is_single_circuit = True
+            circuits = [circuits]
+
+        if self.tracker.active:
+            for c in circuits:
+                self.tracker.update(resources=c.specs["resources"])
+            self.tracker.update(
+                execute_and_derivative_batches=1,
+                executions=len(circuits),
+                derivatives=len(circuits),
+            )
+            self.tracker.record()
+
+        meas = self.execute(circuits, execution_config=execution_config)
+        grad = self.compute_derivatives(circuits, execution_config=execution_config)
+        (results, jacs) = tuple(zip(meas, grad))[0]
+
+        return (results[0], jacs[0]) if is_single_circuit else (results, jacs)
+
+    def supports_jvp(
+        self,
+        execution_config: Optional[ExecutionConfig] = None,
+        circuit: Optional[QuantumTape] = None,
+    ) -> bool:
+        """Whether or not this device defines a custom jacobian vector product."""
+        return False
+
+    def supports_vjp(
+        self,
+        execution_config: Optional[ExecutionConfig] = None,
+        circuit: Optional[QuantumTape] = None,
+    ) -> bool:
+        """Whether or not this device defines a custom vector jacobian product."""
+        return False
+
+    # pylint: disable=unidiomatic-typecheck, unused-argument, too-many-branches
+    def simulate(
+        self,
+        circuit: qml.tape.QuantumScript,
+        rng=None,
+        prng_key=None,
+        debugger=None,
+        interface=None,
+    ) -> Result:
+        """Simulate a single quantum script.
+
+        Args:
+            circuit (QuantumTape): The single circuit to simulate
+            rng (Union[None, int, array_like[int], SeedSequence, BitGenerator, Generator]): A
+                seed-like parameter matching that of ``seed`` for ``numpy.random.default_rng``.
+                If no value is provided, a default RNG will be used.
+            prng_key (Optional[jax.random.PRNGKey]): An optional ``jax.random.PRNGKey``. This is
+                the key to the JAX pseudo random number generator. If None, a random key will be
+                generated. Only for simulation using JAX.
+            debugger (_Debugger): The debugger to use
+            interface (str): The machine learning interface to create the initial state with
+
+        Returns:
+            tuple(TensorLike): The results of the simulation
+
+        Note that this function can return measurements for non-commuting observables simultaneously.
+
+        This function assumes that all operations provide matrices.
+
+        >>> qs = qml.tape.QuantumScript([qml.RX(1.2, wires=0)], [qml.expval(qml.PauliZ(0)), qml.probs(wires=(0,1))])
+        >>> simulate(qs)
+        (0.36235775447667357,
+        tensor([0.68117888, 0.        , 0.31882112, 0.        ], requires_grad=True))
+
+        """
+
+        stim = _import_stim()
+
+        circuit = circuit.map_to_standard_wires()
```

Comment:
`Stim` doesn't support custom-labelled wires, so I used it. Let me know if you would want me to do it some other way.

Reply:
that's quite alright. I just want to double-check that you know what "standard" wire order means, because it's a little quirky. the [second example in the docs](https://docs.pennylane.ai/en/stable/code/api/pennylane.tape.QuantumScript.html#pennylane.tape.QuantumScript.map_to_standard_wires) highlights the quirk best. if you're good with this, then so am I 😄 results should be the same anyway